indico/modules/attachments/blueprint.py

```diff
@@ -66,7 +66,7 @@ def view_func(**kwargs):
 for object_type, prefixes in items:
     for prefix in prefixes:
         if object_type == 'category':
-            prefix = '/category/<categId>' + prefix
+            prefix = '/categories/<category_id>' + prefix
```

Comment:
`/category/` (if there's a conflict with existing endpoints change the old urls to `/category-old/`)